modules/getting-started/proc-techdocs-configure-odf-operator.adoc

```diff
@@ -0,0 +1,35 @@
+[id="proc-techdocs-configure-odf-operator_{context}"]
+
+= Making object storage accessible to containers using the Operator
+
+`ObjectBucket` information is stored in the {product-short} `ObjectBucketClaim` config map and secret. Adding the `ObjectBucket` information to the Operator configuration makes it accessible to the {product-short} container by adding the following environment variables to the container:
+
+* `BUCKET_NAME`
+* `BUCKET_HOST`
+* `BUCKET_PORT`
+* `BUCKET_REGION`
+* `BUCKET_SUBREGION`
+* `AWS_ACCESS_KEY_ID`
+* `AWS_SECRET_ACCESS_KEY`
+
+These variables are then used in the TechDocs plugin configuration.
+
+.Prerequisites
+
+* You have installed {product} on {ocp-short} using the Operator.
+* You have created an `ObjectBucketClaim` CR for storing files generated by TechDocs.
+
+.Procedure
+
+* In the {product-short} `ObjectBucketClaim` CR, enter the name of the {product-short} `ObjectBucketClaim` config map as the value for the `spec.application.extraEnvs.configMaps` field and enter the {product-short} `ObjectBucketClaim` secret name as the value for the `spec.application.extraEnvs.secrets` field. For example:
++
+[source]
```

Comment:
```suggestion
[source,yaml]
```
Suggest for all CRs adding the metadata at the top that comes before `spec` - I think this gives users better context about which CR they're modifying, e.g:

````
apiVersion: objectbucket.io/v1alpha1
kind: ObjectBucketClaim
metadata:
  name: <name>
spec:
````